plugins/modules/pagerduty_alert.py

```diff
@@ -26,24 +27,25 @@
     diff_mode:
         support: none
 options:
-    name:
```

Comment:
It seems that this option was completely removed. That's a breaking change and cannot be merged.

The module needs to keep backwards compatible, so that every task that was correct for the old module still works.

Reply:
Got it.